temporal-sdk/src/main/java/io/temporal/internal/common/WorkflowExecutionHistory.java

```diff
@@ -57,7 +62,7 @@ public WorkflowExecutionHistory(History history, String workflowId) {
     this.workflowId = workflowId;
   }
 
-  public static WorkflowExecutionHistory fromJson(String serialized) {
+  public static io.temporal.common.WorkflowExecutionHistory fromJson(String serialized) {
```

Comment:
Might as well also expose this on the `io.temporal.common.WorkflowExecutionHistory` class unless we are trying to hide it intentionally in internal

Reply:
`static` methods are inherited, so it's already done.

Reply:
Yeah, I can explicitly add a delegate for the doc reason.

Reply:
:1: Maybe for `toJson` too

Reply:
Yeah, I will carry all the public methods as delegates for doc purposes and will write javadocs on them in the public class only.